File: QuizApp/src/QuizApp.BLL/DTO/Url/UrlDetailDTO.cs
using System;
using System.Collections.Generic;
using System.Text;

using QuizApp.BLL.DTO.TestResult;

namespace QuizApp.BLL.DTO.Url
{
    public class UrlDetailDTO
    {
        public int Id { get; set; }

        public int? NumberOfRuns { get; set; }

        public DateTime ValidFromTime { get; set; }

        public DateTime ValidUntilTime { get; set; }

        public string IntervieweeName { get; set; }


        public ICollection<TestResultDetailDTO> TestResults { get; set; }
    }
}

File: frontend/src/app/models/url/url-detail-dto.ts
import { TestDto } from '../test/test-dto';

export interface UrlDetailDto {
    id: number;
    numberOfRuns: number;
    validFromTime: Date;
    validUntilTime: Date;
    intervieweeName: string;
    testId: number;
    test: TestDto;
}
